externals/kyuubi-flink-sql-engine/src/main/scala/org/apache/kyuubi/engine/flink/operation/ExecuteStatement.scala

```diff
@@ -161,7 +161,8 @@ class ExecuteStatement(
       .build(executor)
     val result = executeOperation.invoke[TableResult](sessionId, operation)
     jobId = result.getJobClient.asScala.map(_.getJobID)
-    result.await()
```

Comment:
should we reserve it for Flink 1.14? Or there is no side-effect even we remove it?

Reply:
No side-effect for 1.14, the results are generated immediately after the job is submitted.

Reply:
thanks for confirming, sgtm, pending CI